ui-v2/app/router.js

```diff
@@ -18,6 +18,9 @@ export const routes = {
       show: {
         _options: { path: '/:name' },
       },
+      instance: {
+        _options: { path: '/:name/:id' },
+      },
     },
```

Comment:
Our routing uses a declarative format now. See here for information:

https://github.com/hashicorp/consul/pull/5206

Reply:
It was declarative before too 😛 (but good reminder!)